staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service_unix_test.go

```diff
@@ -228,10 +207,10 @@ func TestIntermittentConnectionLoss(t *testing.T) {
 		t.Fatalf("failed when execute encrypt, error: %v", err)
 	}
 	t.Log("Connected to KMSPlugin")
+	f.CleanUp()
```

Comment:
I would expect `CleanUp` to be unexported since manual cleanup shouldn't be required?

Reply:
It's needed https://github.com/kubernetes/kubernetes/blob/7ebe48f03c60fb49f155e018c712c5e564b50433/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service_unix_test.go#L210-L214 because the test is simulating connection loss and a plugin stop and restart here https://github.com/kubernetes/kubernetes/blob/7ebe48f03c60fb49f155e018c712c5e564b50433/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service_unix_test.go#L232

Reply:
Okay maybe in a follow up we should add a similar test for KMS v2?

Reply:
https://github.com/kubernetes/kubernetes/issues/116999

Reply:
we have [TestIntermittentConnectionLoss](https://github.com/kubernetes/kubernetes/blob/release-1.27/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/grpc_service_unix_test.go#L201) test for KMSv2.